src/error.rs

```diff
@@ -28,6 +28,11 @@ pub(crate) enum Error {
   )]
   DeserializeBody { source: DeserializeBodyError },
 
+  #[snafu(context(false), display("Failed to build embed image: {}", source))]
+  EmbedImageBuild {
```

Comment:
I think I'd keep the error a little more generic:

```suggestion
  #[snafu(context(false), display("Failed to build embed: {}", source))]
  EmbedBuild {
```